lib/logstash/config/mixin.rb

```diff
@@ -444,6 +445,12 @@ def validate_value(value, validator)
             end
 
             result = value.first
+          when :bytes
+            begin
+              result = Filesize.from(value.first).to_i
+            rescue ArgumentError
+              return false, "Unparseable filesize: #{value.first}"
+            end
```

Comment:
I think it would be friendlier to the user to add some units to the message?
What do you think?
```ruby
 "Unparseable filesize: #{value.first}, possible units (KiB, MiB, GiB...)"
```